scripts/convert_html2md.rb

```diff
@@ -4,7 +4,7 @@
 
 parser.banner = <<BANNER
 Usage
-  $ bin/rails r scripts/convert_html2md.rb [options] <fqdn> [id]
+  $ bin/rails r scripts/convert_html2md.rb [options] <fqdn> [public_id]
```

Comment:
`$ bin/rails r scripts/convert_html2md.rb [runner options] -- [options] <fqdn> [public_id]`

Could you fix like above?
I forgot to fix it.